src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -990,6 +990,10 @@ public String getGrpcClientImportName(Interface service) {
     return getNotImplementedString("SurfaceNamer.getGrpcClientImportName");
   }
 
+  public String getClientConfigImportModule(Interface service) {
```

Comment:
Ah, saw an implementation for `getClientConfigPath` that didn't match what was needed. Found that it wasn't used elsewhere in the project. 

This is fixed.